### ISSUE TITLE
(TH10) VoWG speed setting improvements

### ISSUE THREAD

**Issue by inakilbss** [touhouworldcup/thprac#58]
- Change the name from "speed" to "firing interval", "wave delay" or some other name that conveys lower numbers being harder.
- Convey what the minimum interval usually is, currently unclear.
- Add an option to let the interval drop all the way to 0

**Comment by toimine** [comment]
>Convey what the minimum interval usually is, currently unclear.

maybe as dropdown presets? :thinking:

**Comment by 32th-System** [comment]
Second and third option are already resolved. I made it a slider where the minimum value is always 0 and the maximum being the default for whatever difficulty you select

**Comment by 32th-System** [comment]
It doesn't go to 0 in normal gameplay?

**Comment by inakilbss** [comment]
No, it stops decreasing at some point. 24 seconds is the figure I've heard, might be that it drops by 25 in total?
